### PR TITLE
Makes the addon Fastboot 1.0.0 + compatible

### PR DESCRIPTION
**Pull request by linearza**

### CODE REVIEW

**Comment by linearza** [comment]
Hang on - not working correctly when installed, incorrect referencing of the vendor files. Taking a look

File: index.js [hunk 1]
```diff
@@ -14,17 +40,15 @@ module.exports = {
...
+    this.app.import('vendor/jquery.disablescroll.js');
 
     this._super.included.apply(this, arguments);
   },
+
   options: {
     nodeAssets: {
       'tether-shepherd': function() {
         return {
-          enabled: !process.env.EMBER_CLI_FASTBOOT,
```
Comment (rwwagner90) [inline index.js]: You have to also use the fastboot transform with processTree in node assets. Something like this https://github.com/shipshapecode/ember-flatpickr/blob/master/index.js#L20

**Comment by linearza** [comment]
Hmm, not so sure here, having been looking at these type of things all day, but still getting my head around tree merging and the like

**Comment by rwwagner90** [comment]
@linearza you must do the processTree in the nodeAssets, like I linked to an example of.

**Comment by linearza** [comment]
@rwwagner90 - yep, have done that locally, but its only half the solution unfortunately. once you install the addon in an app it loses context of the disablescroll plugin. ill push the processTree change so long though.

File: index.js [hunk 2]
```diff
@@ -53,7 +53,10 @@ module.exports = {
           import: [
             'js/shepherd.js',
             this.theme
-          ]
+          ],
+          processTree(tree) {
```
Comment (rwwagner90) [inline index.js]: This needs to be inside a vendor hash as well, with include and then manually import stuff. Needs a few things to work.

**Comment by rwwagner90** [comment]
@linearza needs some more tweaks to that nodeAssets stuff. Left a comment.

**Comment by linearza** [comment]
@rwwagner90 - sheesh, feels like ive tried every tree in the book - this current version works and builds locally, but unfortunately not once you install it in another app. maybe if you can take a look, im not quite getting my head around it

**Comment by rwwagner90** [comment]
@linearza I really appreciate all the work you put in, but I just made the switch to a different disable scroll lib, which was available on npm, and pushed up the fastboot fix. Please try it out!

**Pull request closed without merging**